### DISCUSSION TITLE
Cant update ani-cli

### DISCUSSION THREAD

**Discussion by nirodhvana** [pystardust/ani-cli#1134] [Q&A] [answered]
Whenever i have to update ani-cli i always use ani-cli -U but now when i run the command it shows:
File ./ani-cli is read-only; trying to patch anyway
patch: **** Can't create temporary file ./ani-cli.oCKpyVA : Permission denied
Can't update for some reason!
Pls help im stuck at version 4.2.2

**Comment by port19x** [comment, marked as answer]
You have `ani-cli` somewhere in your path where the normal user doesn't have write permissions.
Furthermore the output tells you that ani-cli is read-only, which is a bad sign.

You could try `sudo ani-cli -U` or alternatively remove ani-cli and reinstall, possibly via one of our native packages
